<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPerson</class>
 <widget class="QDialog" name="AddPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прізвище, ім'я:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_iban">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IBAN:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_bank">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Банк:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_edrpoy">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ЄДРПОУ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ipn">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ІПН:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Адреса:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_number">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Номер:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_email">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ел. пошта:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_password">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="line_name">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прізвище</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_iban">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IBAN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_bank">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Банк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_edrpoy">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ЄДРПОУ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_ipn">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ІПН</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_address">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Адреса</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_number">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Номер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_email">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ел. Пошта</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_password">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Додати</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reject_3">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_reject_3</sender>
   <signal>clicked()</signal>
   <receiver>AddPerson</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
